LEGO1/lego/legoomni/include/legocarbuildpresenter.h

```diff
@@ -83,7 +83,7 @@ class LegoCarBuildAnimPresenter : public LegoAnimPresenter {
 	LegoTreeNode* FindNodeByName(LegoTreeNode* p_treeNode, const LegoChar* p_name);
 	void AddPartToBuildByName(const LegoChar* p_name);
 	void RotateAroundYAxis(MxFloat p_angle);
-	MxBool FUN_10079c30(const LegoChar* p_name);
+	MxBool CurrentPartToPlace(const LegoChar* p_name);
```

Comment:
```suggestion
	MxBool IsNextPartToPlace(const LegoChar* p_name);
```

I'm wondering if this name makes it easier to understand, what this function does.